mabwiser/linear.py

```diff
@@ -57,21 +58,51 @@ def predict(self, x):
 
         # Scale
         if self.scaler is not None:
-            x = x.reshape(1, -1)
-            x = self.scaler.transform(x.astype('float64')).reshape(-1)
+            x = self._scale_predict_context(x)
 
         # Calculate default expectation y = x * b
         return np.dot(x, self.beta)
 
+    def _scale_predict_context(self, x):
```

Comment:
can we add comments to this helper functions.. what does reshape(1, -1) does, transforms? floats?